### PR TITLE
Allow configurable role for ChoiceGroup container DIV, default to "application"

### LINKED ISSUE #6479: Role='application' around ChoiceGroup; bug or feature?
Around ChoiceGroup there is a wrapper with role='application'. When testing for accessibility-issues my screen reader software reads this as _"Your are in an application"_. 

The radio buttons are then read correctly, but I wonder if the role really is necessary? Every document I've found online says something like: _"Use role='application' with care"._

I cannot find a way to remove or disable the role='application' from the ChoiceGroup component.

### PR DESCRIPTION
**Pull request by KevinTCoughlin**
#### Pull request checklist

- [] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

**Note:** Removing linked issue since we want to explore removing this `role="application"` entirely if the JAWS issue is resolved.

#### Description of changes

Allows a developer to override the `role` attribute of the containing `<div>` for `<ChoiceGroup />`. Defaults to `role="application"` to address a `JAWS` compatibility issue.

Adds unit tests to check default and override behavior.

I'm a tad hesitant on this change only because of the in-comment referencing the JAWS limitation so this is a "Use at your own risk" prop regarding accessibility.

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.test.tsx [hunk 1]
```diff
@@ -152,4 +146,20 @@ describe('ChoiceGroup', () => {
     expect(extraAttributeGetter(0)).toEqual('auto1');
     expect(extraAttributeGetter(1)).toBeNull();
   });
+
+  it('has role="application" by default on the containing element', () => {
```
Comment (cliffkoh) [inline packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.test.tsx]: The comment that is in the code providing context should be here in the unit test as well.

**Comment by cliffkoh** [comment]
Looks like a change it would be great to have either @srideshpande or one of the accessibility experts sign off on.

**Comment by dzearing** [comment]
@cschlechty role=application use here. I'm not sure it's the right scenario. I know historically we've been very reluctant to use this.

**Comment by KevinTCoughlin** [comment]
> 
> 
> @cschlechty role=application use here. I'm not sure it's the right scenario. I know historically we've been very reluctant to use this.

We've had some follow-up in https://github.com/OfficeDev/office-ui-fabric-react/issues/6479#issuecomment-426773980

I'm thinking we can merge this so devs can opt-in to the role removal. Then I can see if the JAWS issue is resolved so the role can be removed completely in a subsequent PR.

**Comment by cschlechty** [comment]
I'm ok with merging this.

**Pull request merged**